web-frontend/src/main.tsx

```diff
@@ -25,7 +25,7 @@ import SendCoins from './pages/TransferCoins/SendCoins.tsx';
 import Home from './pages/Home/Home.tsx';
 import ReceiveCoins from './pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx';
 import Settings from './pages/Settings/Settings.tsx';
-import SettingsEdit from './pages/Settings/SeetingsEdit.tsx';
+import SettingsEdit from './pages/Settings/SettingsUpdate.tsx';
```

Comment:
SettingsEdit -> SettingsEdit ✅ 

now you are changing to:

SettingsEdit -> SettingsUpdate 🚫 

🤯

Reply:
Oh sorry I miss that

Reply:
Done